sdk/azidentity/environment_credential.go

```diff
@@ -25,29 +25,36 @@ type EnvironmentCredentialOptions struct {
 // EnvironmentCredential authenticates a service principal with a secret or certificate, or a user with a password, depending
 // on environment variable configuration. It reads configuration from these variables, in the following order:
 //
-// Service principal:
-// - AZURE_TENANT_ID: ID of the service principal's tenant. Also called its "directory" ID.
-// - AZURE_CLIENT_ID: the service principal's client ID
-// - AZURE_CLIENT_SECRET: one of the service principal's client secrets
+// Service principal with client secret
 //
-// Service principal with certificate:
-// - AZURE_TENANT_ID: ID of the service principal's tenant. Also called its "directory" ID.
-// - AZURE_CLIENT_ID: the service principal's client ID
-// - AZURE_CLIENT_CERTIFICATE_PATH: path to a PEM or PKCS12 certificate file including the private key. The
-//   certificate must not be password-protected.
+// AZURE_TENANT_ID: ID of the service principal's tenant. Also called its "directory" ID.
 //
-// User with username and password:
-// - AZURE_CLIENT_ID: the application's client ID
-// - AZURE_USERNAME: a username (usually an email address)
-// - AZURE_PASSWORD: that user's password
-// - AZURE_TENANT_ID: (optional) tenant to authenticate in. If not set, defaults to the "organizations" tenant, which
-//   can authenticate only Azure Active Directory work or school accounts.
+// AZURE_CLIENT_ID: the service principal's client ID
+//
+// AZURE_CLIENT_SECRET: one of the service principal's client secrets
+//
+// Service principal with certificate
+//
+// AZURE_TENANT_ID: ID of the service principal's tenant. Also called its "directory" ID.
+//
```

Comment:
It seems slightly harder to see that some of these options are grouped since everything has an extra newline after it (IMO).

Reply:
Ah, I just noticed your PR description. Ignore this one.